src/operator/tensor/matrix_op-inl.h

```diff
@@ -690,9 +701,10 @@ inline bool SliceOpShape(const nnvm::NodeAttrs& attrs,
   CHECK_EQ(in_attrs->size(), 1U);
   CHECK_EQ(out_attrs->size(), 1U);
   const TShape& dshape = (*in_attrs)[0];
-  if (dshape.ndim() == 0 || dshape.Size() == 0) return false;
```

Comment:
Since we are removing one of the checks, please add
```cpp
return !shape_is_none(dshape) && !shape_is_none(oshape)
```
at the end of this function.

Reply:
Added